add note about how to learn about color (#1929)
primer/css: docs/content/utilities/colors.mdx

  Please note Primer CSS v18 has changed the naming of these color classes. Check the <a href="/css/support/v18-migration">migration guide</a> to make sure your app is up to date.
</Note>

> Use color utilities to apply color to the background of elements, text, and borders. These utilities should be used for specific color usage that is not provided by a component. To learn more about how color is used and how to appropriately apply it to your project, read the [color documentation](https://primer.style/design/foundations/color) in the [interface guidelines](https://primer.style/design/).

Reviewer: ```suggestion
Use color utilities to apply color to the background of elements, text, and borders. These utilities should only be used where color is not provided by a component. To learn more about how to apply color, read the [color documentation](https://primer.style/design/foundations/color) in the [interface guidelines](https://primer.style/design/).
```
